table/metadata.go

```diff
@@ -743,6 +753,11 @@ func (b *MetadataBuilder) SetLastUpdatedMS() *MetadataBuilder {
 	return b
 }
 
+func (b *MetadataBuilder) SetNextRowID(nextRowID int64) *MetadataBuilder {
+	b.nextRowID = &nextRowID
+	return b
+}
```

Comment:
do we need a setter here? On what occasion would we want to set this from the outside? 

It probably makes sense to not expose this to library users. My current understanding is that we only need to increment this in AddSnapshot based on the number of added rows in the snapshot.